### ISSUE TITLE
App crash on Android 13

### ISSUE THREAD

**Issue by Generator** [Kunzisoft/KeePassDX#1589]
**Describe the bug**

When trying to open an entry, the app crashes. One time even crashed the whole system.

**To Reproduce**

1. Enable autofill
2. Open an entry

**KeePass Database**

 - Created with: KeePassXC
 - Version: ??
 - Location: local
 - File provider (`content://` URI): ``content://com.android.externalstorage.documents/document/primary%3ADocuments%2FKeePass%2Fkpdb.kdbx``
 - Size: 1.3MB
 - Contains attachment: Yes

**KeePassDX:**

 - Version: 3.5.1
 - Build: F-Droid
 - Language: Portuguese

**Android:**

 - Device: Xiaomi Redmi Note 8T
 - Version: 13 (CrDroid 9.6)

**Additional context**

 - Browser for Autofill: Fulguris

Already tried with clean install.

Logs:
[keepassdx.log.txt](https://github.com/Kunzisoft/KeePassDX/files/12002371/keepassdx.log.txt)
Autolog from CrDroid: https://paste.crdroid.net/RnLYTe

**Comment by J-Jamet** [comment]
I'm not able to reproduce the bug, so I've set an exception catch when unbinding the service. Let me know if that solves the problem for you with version 4.0.0.

**Comment by Generator** [comment]
I think that could be also related with the ROM version, but need more testing. 
KeePass2Android don't have any issues.

I'll try with a clean install see if solves

**Comment by v2302** [comment]
Hello, I have experienced the same issue, log results are similar too. In my case, it not only caused crashes, but also reboots.

Device Information:
- Android 13
- crDroid 9.8 and LineageOS 20 (Tested on both)
- KeePassDX v3.5.1 (F-Droid)

Anyway, I am pleased to report that the beta version (4.0.0) have fixed the issue. I have tested it on both crDroid 9.8 and LineageOS 20 (both Android 13) and I am no longer able to reproduce the bug.
